state-synchronizer/src/tests/helpers.rs

```diff
@@ -131,11 +131,14 @@ impl ExecutorProxyTrait for MockExecutorProxy {
         limit: u64,
         target_version: u64,
     ) -> Result<TransactionListWithProof> {
+        let start_version = known_version
```

Comment:
Weird that we have to fix this in the test code.. I'm guessing it's because the fuzzer issue found was actually here (in test code) as opposed to non-test code, but the issue that it highlighted is more generic than just test code? 😄

Reply:
yeah I wrote that in the report, the issue was in the test code, but looking at the real code it's there too : (